app/models/school.rb

```diff
@@ -130,6 +130,10 @@ def allocation_for_type!(device_type)
     device_allocations.find_by_device_type!(device_type)
   end
 
+  def is_eligible_to_order?
```

Comment:
Note: I moved this out of the `private` methods as it's now being used outside this model.